### PR TITLE
Documenting contribution process

### PR DESCRIPTION
**Pull request by jan-gogo**
This pull request adds documentation about the contribution process.

- [x] Estimations
- [x] Workflow
- [x] Buddy system
- [x] Environment setup

### CODE REVIEW

**Comment by piotradamczyk5** [comment]
Looks good to me 👍

**Comment by adamfilipow92** [comment]
👍

File: docs/onboarding/1_environment_setup.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+### Env config
+Bunch of useful exports. You can paste them to your `.bashrc`
+```bash
+FLANK_REPO="type path to your local flank repository"
+export PATH=$PATH:$HOME/$FLANK_REPO/flank/test_runner/bash
+export PATH=$PATH:$HOME/$FLANK_REPO/flank/test_app/bash
+export PATH=$PATH:$HOME/Library/Android/sdk/platform-tools
+export PATH=$PATH:$HOME/Library/Python/2.7/bin
+#export PATH=$PATH:$HOME/"path to your local gcloud repository"/gcloud_cli/google-cloud-sdk/bin
```
Comment (pawelpasterz) [inline docs/onboarding/1_environment_setup.md:16]: Do we want to have this commented?

File: docs/onboarding/1_environment_setup.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
+# Environment setup
+This document may be incomplete now or in the future, so if you faced any problems ask the team for help.
+
+### Mac
+1. Install a brew, it's not mandatory but may be convenient for installing other software.
+1. Currently, the zsh is default shell on a mac. If you prefer bash use `chsh -s /bin/bash`.
```
Comment (pawelpasterz) [inline docs/onboarding/1_environment_setup.md:6]: `is the default`

File: docs/onboarding/1_environment_setup.md (hunk 1 above)
Comment (jan-gogo) [reply docs/onboarding/1_environment_setup.md:16]: yup

File: docs/onboarding/1_environment_setup.md (hunk 2 above)
Comment (jan-gogo) [reply docs/onboarding/1_environment_setup.md:6]: nope

Comment (bootstraponline) [reply docs/onboarding/1_environment_setup.md:6]: image

I think pawel is correct. Grammerly.com is helpful for issues like this.

I'd write it like this:
> Zsh is the default shell on macOS.

Comment (jan-gogo) [reply docs/onboarding/1_environment_setup.md:6]: Right I misunderstood the comment. I will fix it in another PR.

**Pull request merged**